peloton/common.py

```diff
@@ -3,26 +3,55 @@
 dev team in keeping the repos healthy.
 """
 
-# TODO: use the SciTools/peloton user group instead.
-peloton_logins = [
-    "bjlittle",
-    "bsherratt",
-    "ESadek-MO",
-    "codecov[bot]",
-    "HGWright",
-    "jamesp",
-    "lbdreyer",
-    "pp-mo",
-    "rcomer",
-    "stephenworsley",
-    "tkknight",
-    "trexfeathers",
-    "wjbenfold",
-    "dependabot[bot]",
-    "github-actions[bot]",
-    "pre-commit-ci[bot]",
-    "scitools-ci[bot]",
-]
+import json
+from pathlib import Path
+from subprocess import run
+from typing import List
+
+
+csv_path = Path(__file__).parent / "peloton.csv"
+
+
+def _get_peloton_logins() -> List[str]:
+    query = """
+        query { 
+          organization(login: "SciTools") { 
+            team(slug: "Peloton") {
+              members(first: 100) {
+                nodes {
+                  login
+                }
+              }
+            }
+          }
+        }
+    """
+    print(query)
```

Comment:
Was this used for testing?

Reply:
7441113bd1a1e0ee51252947fbbb735a0d7319fe